Ntuplizer/python/templates/eventBranches.py

```diff
@@ -10,6 +10,7 @@
         type1_pfMETPhi = cms.string('type1_pfMETPhi'),
         ),
     vFloats = cms.PSet(
+        lheWeights = cms.vstring('lheWeights'),
```

Comment:
These branches are included for data. Better to move this to `eventGenBranches` below, or maybe to a separate `PSet` to allow configuration of which weights to include. It's also presumably needed in the gen ntuples; I'm not sure whether we'd want it in both in that case.